content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md

```diff
@@ -0,0 +1,291 @@
+### Importing External Files using Webpack Loaders: The Beginners Guide 
+In this article,  we will learn about Webpack and how to use Webpack loaders to import external files while making use of Webpack. 
```

Comment:
Along with this, give 1 - 2 lines of what webpack loaders are, where you use it, or how it works